engines/stark/ui/world/topmenu.cpp

```diff
@@ -41,9 +41,9 @@ TopMenu::TopMenu(Gfx::Driver *gfx, Cursor *cursor) :
 	_position = Common::Rect(Gfx::Driver::kOriginalWidth, Gfx::Driver::kTopBorderHeight);
 	_visible = true;
 
-	_inventoryButton = new Button("Inventory", StaticProvider::kInventory, Common::Point(32, 2));
-	_optionsButton = new Button("Options", StaticProvider::kDiaryNormal, Common::Point(560, 2));
-	_exitButton = new Button("Quit", StaticProvider::kQuit, Common::Point(608, 2));
+	_inventoryButton = new Button("Inventory", StaticProvider::kInventory, Common::Point(32, 2), Common::Point(64, 20));
+	_optionsButton = new Button("Options", StaticProvider::kDiaryNormal, Common::Point(560, 2), Common::Point(523, 20));
```

Comment:
Using the top left position of the hint for the options and quit buttons is going to be an issue when we support loading the labels from the translation files. The non-english text will not have the same length. The top right position needs to be used.